packages/read-model/easy-coding-standard.yml

```diff
@@ -0,0 +1,21 @@
+imports:
+    - { resource: '%vendor_dir%/shopsys/coding-standards/easy-coding-standard.yml', ignore_errors: true }
+
+services:
+    # this package is meant to be extensible using class inheritance, so we want to avoid private visibilities in the model namespace
```

Comment:
maybe this comment can be written in commit message

Reply:
Like this, it's consistent with other packages. I prefer the comment here to be clear on the first sight.